src/index.html

```diff
@@ -4,11 +4,34 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="style.css">
+    <!-- Bootstrap -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <!-- Font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <title>To Do List</title>
 </head>
 <body>
-    <h1>Hello Webpack</h1>
+    
+    <div class="container">
+        <div id="tasks-container" class="col-12 col-md-8 m-auto">
+            <h1 class="text-center">To Do List</h1>
+            <div class="shadow-sm">
+                <ul id="main-tasks" class="list-group">
+                    <li class="list-group-item"><div class="d-flex justify-content-between"><p class="my-auto">Today's To Do</p><i class="fa-solid fa-arrows-rotate my-auto"></i></div></li>
+                    <li class="list-group-item"><div class="d-flex justify-content-between"><input id="input-task" type="text" placeholder="Add your list..."> <i class="fa-solid fa-caret-right my-auto"></i></div></li>
+                    <!-- <li class="list-group-item">An item</li>
+                    <li class="list-group-item">A second item</li>
+                    <li class="list-group-item">A third item</li>
+                    <li class="list-group-item">A fourth item</li>
+                    <li class="list-group-item">And a fifth one</li> -->
```

Comment:
- [ ] Kindly make sure to remove the commented codebase in your codebase :wink:

Reply:
Thanks for the advice. I will remove it